packages/terra-base/src/terra-dev-site/doc/base/ChangeLog.2.doc.mdx

```diff
@@ -0,0 +1,3 @@
+import ChangeLog from '../../../../CHANGELOG.md';
```

Comment:
Should this render the Badge as well?

Reply:
Updated here https://github.com/cerner/terra-core/pull/2803/commits/4e5536321edd517b2095c11caf3ed29c8e5ee657.